apis/nucleus/src/components/listbox/utils/frequencyMaxUtil.js

```diff
@@ -1,12 +1,4 @@
-const escapeField = (field) => {
-  if (!field) {
```

Comment:
This line the new file check `if (!field || field === ']')`. I guess it is fine to use new file logic. @T-Wizard

Reply:
OK.
if I remember correctly `field === ']'` was from engine making it impossible to use a field with the name `]`
but now it looks like it works both with and without escape